src/test/java/jrm/user/DeleteGuestUsersTest.java

```diff
@@ -0,0 +1,70 @@
+package jrm.user;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.ValidatableResponse;
+import jrm.commonclasses.CommonTest;
+import jrm.dao.UserRepository;
+import jrm.dto.UserDto;
+import jrm.service.UserAuthApiService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Description;
+
+import java.sql.Timestamp;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+public class DeleteGuestUsersTest implements CommonTest {
+
+    @Autowired
+    UserAuthApiService userAuthApiService;
+
+    @Autowired
+    UserRepository userRepository;
+
+
+    @Test
+    @Description("manual deleting for old guest user")
+    void deleteGuestUsersTest(){
```

Comment:
What about bound scenarios when user has been created 23 hours ago or 25 hours ago? Seems like it should be added.

Reply:
What a great advice! Unfortunately it was not came up on my mind. I'll try to implement it.

Reply:
Added scenarios according to your edits.